src/utils.jl

```diff
@@ -41,7 +41,7 @@ function create_callback(
         callbackstate = (;
             θmin = θ, loss_min = eltype(θ)(Inf), lhist_val = [], lhist_train = []),
         nunroll = nothing, batch_size = nothing, rng = Random.Xoshiro(123), do_plot = true,
-        plot_train = true, plot_every = 10)
+        plot_train = true, plot_every = 10, device = identity)
```

Comment:
can we make identity the default choice? or maybe even avoid doing `identity()` and only do `dev()` when it is on gpu